src/VisualStudio/CSharp/Impl/CSharpExtension.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.VisualStudio.Extensibility;
+using Microsoft.VisualStudio.Extensibility.Commands;
+using Microsoft.VisualStudio.LanguageServices.CSharp.Interactive;
+using Microsoft.VisualStudio.Shell;
+
+namespace Microsoft.VisualStudio.LanguageServices;
+
+[VisualStudioContribution]
+internal sealed class CSharpExtension : Extension
+{
+    public override ExtensionConfiguration ExtensionConfiguration => new()
+    {
+        RequiresInProcessHosting = true,
+    };
+
+    // Can't reference non-constant Shell.VsMenus members from CommandConfiguration: 
+    private static readonly Guid s_guidSHLMainMenu = new(0xd309f791, 0x903f, 0x11d0, 0x9e, 0xfc, 0x00, 0xa0, 0xc9, 0x11, 0x00, 0x4f);
```

Comment:
This should have a link to a bug and also a `<seealso>` link to the VsMenus item it is copied from

Reply:
Not a bug - VS SDK just doesn't support it. I can add the link to the item.